spec/forms/policy_form_spec.rb

```diff
@@ -120,4 +119,11 @@ def set_organisation_attributes(lead = [], supporting = [])
       expect(policy.organisation_content_ids).to eql [1,2,3,4]
     end
   end
+
+private
+  def set_organisation_attributes(lead = [], supporting = [])
```

Comment:
Can't say I've ever seen the need for private methods in specs. I'd rather this were right up the top as part of the setup stage so it's known upfront to the reader what the behaviour is here.